src/components/ModeMenu.js

```diff
@@ -0,0 +1,14 @@
+import ModeMenuButton from "./ModeMenuButton";
+
+const ModeMenu = (onButtonClick) => {
+    const section = document.createElement('div');
+    section.classList.add('modeMenu');
+
+    ['Characters', 'Quotes', 'Deaths'].map((buttonName)=>{
+      let newButton = ModeMenuButton(buttonName,onButtonClick);
```

Comment:
Zawsze stawiaj `const` To podstawa!
`let `tylko jak nie masz innego wyjścia.

```suggestion
      const newButton = ModeMenuButton(buttonName,onButtonClick);
```